NotifyIconWin32/NotificationAreaIcon.cpp

```diff
@@ -57,6 +57,7 @@ void NotificationAreaIcon::RefreshIcon() {
 	try {
 		Shell_NotifyIcon(NIM_ADD, _icon_data);
 		Shell_NotifyIcon(NIM_MODIFY, _icon_data);
+		SetVersion();
```

Comment:
We weren't doing this previously, which meant we hadn't opted in to receive the WM_CONTEXTMENU event.

Issue described here: https://stackoverflow.com/questions/8404435/notify-icon-receives-wm-lbuttondblclk-but-not-wm-contextmenu